lib/command_parse.y

```diff
@@ -505,6 +506,12 @@ terminate_graph (CMD_YYLTYPE *locp, struct parser_ctx *ctx,
     while (ctx->docstr && ctx->docstr[1] != '\0')
       zlog_err ("%s", strsep(&ctx->docstr, "\n"));
     zlog_err ("----------");
+    /*
+     * The backtrace here will cause our test system to go belly
+     * up and fail the test so hopefully this will not get into
+     * our actual code base
+     */
+    assert(0);
```

Comment:
Can we add a message here and not just a simple assert? Will make it much clearer
ie. `assert(0, "Backtrace found")`

Reply:
looking at the output of the code immediately before the assert it becomes abundantly clear imo what the problem is.  I am not sure it's really needed.

Reply:
here's the output:
```sharpd@eva ~/f/t/bgpd (tests_need_to_be_stricter)> ./test_peer_attr
2023/06/30 12:45:33 [K2CCG-5Y7ZJ] Excessive docstring while parsing 'no set as-path replace [<any|ASNUM>] [<ASNUM>$configured_asn]'
2023/06/30 12:45:33 [W7ENN-K2SVA] ----------
2023/06/30 12:45:33 [WCW75-6TZPF] Define the configured AS number
2023/06/30 12:45:33 [W7ENN-K2SVA] ----------
2023/06/30 12:45:33 lib/command_parse.y:514: terminate_graph(): assertion (0) failed
fish: Job 1, './test_peer_attr' terminated by signal SIGABRT (Abort)
sharpd@eva ~/f/t/bgpd (tests_need_to_be_stricter) [SIGABRT]> 
````

If you can't figure that out.  I can't help you

Reply:
I pulled this commit out because I could not make it work in conjunction with any sort of make check